WalletWasabi.Fluent/ViewModels/Wallets/Send/OptimisePrivacyViewModel.cs

```diff
@@ -1,9 +1,56 @@
+using System.Collections.ObjectModel;
 using WalletWasabi.Fluent.ViewModels.Navigation;
 
 namespace WalletWasabi.Fluent.ViewModels.Wallets.Send
 {
 	[NavigationMetaData(Title = "Optimise your privacy")]
 	public partial class OptimisePrivacyViewModel : RoutableViewModel
 	{
+		[AutoNotify] private ObservableCollection<PrivacySuggestionControlViewModel> _privacySuggestions;
+		[AutoNotify] private PrivacySuggestionControlViewModel _selectedPrivacySuggestion;
+
+		public OptimisePrivacyViewModel()
+		{
+			_privacySuggestions = new ObservableCollection<PrivacySuggestionControlViewModel>();
+
+			_privacySuggestions.Add(new PrivacySuggestionControlViewModel
+			{
+				Title = "1.13 BTC",
+				Caption = "Less",
+				Benefits = new[]
+				{
+					"Send 0.01% Less",
```

Comment:
I am just wondering... Wouldn't it be nice if the users know how much more/less will they send in USD?

Reply:
we might move this info to a tooltip.
maybe .